tw-tasks-core/src/main/java/com/transferwise/tasks/TasksProperties.java

```diff
@@ -255,6 +253,14 @@
    */
   private boolean paranoidTasksCleaning = false;
 
+  /**
+   * Copies data to old data field as well.
+   *
+   * <p>When we are rolling out a new version, nodes with old version need to be able to process new tasks.
```

Comment:
This only applies to older versions being updated to the current one, right? I think the comment could be more specific.